src/root/views/Search/index.tsx

```diff
@@ -117,153 +114,127 @@ function Search(props: Props) {
       (item, key) => ({
         key: key as ResultKeys,
         totalItems: item.length,
-        value: sum(item.slice(0, MAX_VIEW_PER_SECTION).map(scoreSelector))
+        value: Math.max(0, ...(item.map((i) => i.score))),
       }),
     );
 
-  const keysOrdering: {
-    [key in ResultKeys]: number
-  } = {
-    regions: 0,
-    countries: 1,
-    provinces: 2,
-    emergencies: 3,
-    appeals: 3,
-    projects: 3,
-    surgeAlerts: 3,
-    surgeDeployments: 3,
-    fieldReports: 3,
-  };
-  const sortedScoreList = scoreList.sort((a, b) => (
-    keysOrdering[a.key] - keysOrdering[b.key] || b.value - a.value
-  ));
-
-  const isEmpty = scoreList.every((score) => score.totalItems === 0);
-
-  return [
-    resultsMap,
-    componentMap, sortedScoreList,
-    isEmpty,
-  ];
-}, [searchResponse]);
+    const keysOrdering: {
+      [key in ResultKeys]: number
+    } = {
+      regions: 0,
+      countries: 1,
+      provinces: 2,
+      emergencies: 3,
+      appeals: 3,
+      projects: 3,
+      surgeAlerts: 3,
+      surgeDeployments: 3,
+      fieldReports: 3,
+    };
+    const sortedScoreList = scoreList.sort((a, b) => (
+      keysOrdering[a.key] - keysOrdering[b.key] || b.value - a.value
+    ));
 
-const ActiveComponent = activeView ? componentMap[activeView] : undefined;
-const redirectSearchString = isDefined(debouncedSearchString) ? `?${URL_SEARCH_KEY}=${window.encodeURI(debouncedSearchString)}` : undefined;
-const currentSearchString = window.location.search;
+    const isEmpty = scoreList.every((score) => score.totalItems === 0);
```

Comment:
here `scoreList` is an array of search items. We should always name variables in a way that is concise and contextual. Let's rename this accordingly. 
If you have free time please explore: https://github.com/topics/naming-cheatsheet